export function testErrorThrow() {
    console.log("Error From test.js");
}


export default testErrorThrow;